### DISCUSSION TITLE
Minimum macOS version (binary release vs. source code)

### DISCUSSION THREAD

**Discussion by elder-n00b** [jesseduffield/lazygit#1427] [General]
For information: the readme says _"For Windows, Mac OS(**10.10+**) or Linux, you can download a binary release here"_ but current release 0.28.2 fails on macOS 10.11.6 with `dyld: Symbol not found: _clock_gettime` which indicates 10.12+ . 

**Compiling from source with go 1.15 does work though**. IIRC go1.15.5 is the last one to support macOS before 10.12 .

And by the way... lazygit is very nice!

**Comment by jesseduffield** [comment]
Thanks, I've gone ahead and updated the readme. And good to hear you're liking lazygit!
